cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -22,6 +22,7 @@
 
 // Note: The <cuda/std/*> versions are used in order for Jitify to work with our fixed_point type.
```

Comment:
Should this comment be copied to the other files where these includes occur?

Reply:
I don't know -- this comment precedes this PR.